apps/studio/src/features/editing-experience/components/form-builder/renderers/controls/JsonFormsBooleanControl.tsx

```diff
@@ -25,14 +25,15 @@ export function JsonFormsBooleanControl({
   path,
   description,
   schema,
+  required,
 }: ControlProps): JSX.Element {
   if (schema.const !== undefined) {
     return <></>
   }
 
   return (
     <Box>
-      <FormControl isRequired>
+      <FormControl isRequired={required}>
```

Comment:
question(non-blocking): hmm I recall discussing with design on this before and it seems like having the words "(optional)" for a toggle from a design perspective seemed a little weird to users, hence it is set to required. I think the `defaultValue` should be able to ensure that a value is set, to fulfil the FormControl's requirement?